stack.go

```diff
@@ -65,37 +65,27 @@ type frame uintptr
 
 // get returns a human readable stack frame.
 func (f frame) get() StackFrame {
-	frame := StackFrame{
-		Name: "unknown",
-		File: "unknown",
-	}
-
 	pc := uintptr(f) - 1
-	fn := runtime.FuncForPC(pc)
-	if fn != nil {
-		name := fn.Name()
-		i := strings.LastIndex(name, "/")
-		name = name[i+1:]
-		file, line := fn.FileLine(pc)
-
-		frame = StackFrame{
-			Name: name,
-			File: file,
-			Line: line,
-		}
-	}
+	frames := runtime.CallersFrames([]uintptr{pc})
```

Comment:
The golang team recommends using `CallersFrames` instead of `FuncForPC`

Reply:
do you know why? or can you link to the recommendation?

Reply:
we have an issue for it #54